sections/errorhandling/operationalvsprogrammererror.spanish.md

```diff
@@ -38,21 +72,21 @@ De la documentación oficial de Node.js
 
- > …Por la misma naturaleza de cómo throw funciona in JavaScript, casi nunca hay una forma segura de "seguir dónde lo dejaste", sin generar fugas de referencias, o sin crear algún otro tipo de estado frágil indefinido. La forma más segura de responder a un error lanzado is apagar el proceso. Claro, en un web server normal, podrías tener muchas conexiones abiertas, y no es rasonable cerrarlas abruptamente porque un error fue disparado por alguna otra persona. El mejor enfoque is enviar una respuesta de error a la petición que disparó el error mientras dejas las otras terminar en su tiempo normal, y parar de escuchar nuevas peticiones en ese worker.
+ > …Por la misma naturaleza de cómo throw funciona in JavaScript, casi nunca hay una forma segura de "seguir dónde lo dejaste", sin generar fugas de referencias, o sin crear algún otro tipo de estado frágil indefinido. La forma más segura de responder a un error lanzado es apagar el proceso. Claro, en un servidor web normal, podrías tener muchas conexiones abiertas, y no es razonable cerrarlas abruptamente porque un error fue disparado por alguna otra persona. El mejor enfoque es enviar una respuesta de error a la petición que disparó el error mientras dejas las otras terminar en su tiempo normal, y parar de escuchar nuevas peticiones en ese worker.
 
 ### Cita de blog: "De otra manera, pones en riesgo el estado de tu aplicación"
 
 Del blog, debugable.com clasificado 3ro para las palabras clave “Node.js uncaught exception””
 
  > …So, unless you really know what you are doing, you should perform a graceful restart of your service after receiving an “uncaughtException” exception event. Otherwise, you risk the state of your application, or that of 3rd party libraries to become inconsistent, leading to all kinds of crazy bugs…
 
- > …Entonces, a manos que ralmente sepas lo que estás haciendo, deberías ejecutar un reinicio controlado de tu servicio luego de recibir un evento de excepción "uncaughtException". De otra manera, pones en riesgo el estado de tu aplicación, o que una biblioteca de terceros se vuelvan inconsistentes, llevando a todo tipo de errores locos…
+ > …Entonces, a menos que realmente sepas lo que estás haciendo, deberías ejecutar un reinicio controlado de tu servicio luego de recibir un evento de excepción "uncaughtException". De otra manera, pones en riesgo el estado de tu aplicación, o que una biblioteca de terceros se vuelva inconsistente, llevando a todo tipo de bugs locos…
```

Comment:
a manos = to hands -> a menos = unless
ralmente is typo
`vuelvan inconsistentes` -> `vuelva inconsistente` changing to plural since talking about **one** library
`errores` could be mistaken for errors (and ofc errors !== bugs) a translation could be `defectos` but most ppl use the word `bugs` untranslated